setup/install_gatling/50_tune/10_puppet_infrastructure_tune.rb

```diff
@@ -45,12 +43,7 @@ def puppet_infrastructure_tune
     ].join(" ")
     on master, commit
 
-    # re-run the agent up to MAX_AGENT_RUNS attempts
-    retry_params = { max_retries: MAX_AGENT_RUNS,
-                     retry_interval: 2,
-                     desired_exit_codes: [0] }
-
-    retry_on(master, puppet("agent", "-t"), retry_params)
+    run_agent_until_no_change(master)
```

Comment:
This actually fails:
```
NoMethodError: undefined method `each' for #<Unix::Host:0x00007f49ea763f78>
/home/centos/gplt/647/gatling-puppet-load-test/setup/helpers/perf_helper.rb:731
/home/centos/gplt/647/gatling-puppet-load-test/setup/install_gatling/50_tune/10_puppet_infrastructure_tune.rb:46
```

For some reason I thought beaker hosts could handle this; perhaps this needs to be called as:
```
run_agent_until_no_change([master])
```
?

Reply:
:man_facepalming: sorry @billclaytor . This has been fixed.